packages/opentelemetry-core/src/trace/instrumentation/BasePlugin.ts

```diff
@@ -21,6 +21,7 @@ export abstract class BasePlugin<T> implements Plugin<T> {
   protected _moduleExports!: T;
   protected _tracer!: Tracer;
   protected _logger!: Logger;
+  supportedVersions?: string[];
```

Comment:
`protected` WDYT?

Reply:
if I change to `protected` I can't do `plugin.supportedVersions` in PluginLoader

Reply:
You are right, I thought we were doing this in individual plugin. Pls ignore this comment.